reactor-netty-http/src/test/java/reactor/netty/resources/PooledConnectionProviderDefaultMetricsTest.java

```diff
@@ -388,6 +388,8 @@ private void testIssue3060(ConnectionProvider provider) throws Exception {
 	/* https://github.com/reactor/reactor-netty/issues/3519 */
 	@Test
 	public void testConnectionProviderDisableAllBuiltInMetrics() throws Exception {
+		REGISTRY.clear();
+
```

Comment:
Although not related to the purpose of this PR, handling has been implemented for failing tests. 
To prevent interference with other tests, the `metrics` are initialized.